code/src/App.js

```diff
@@ -1,9 +1,67 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import MessageInput from './components/MessageInput';
+import MessageList from './components/MessageList';
+import Loader from './components/Loader';
+import { MESSAGES_URL } from './urls';
+
+import './components/Styles.scss';
 
 export const App = () => {
+  const [messages, setMessages] = useState([]);
+  const [isLoading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetchMessages();
+    setInterval(fetchMessages, 5000);
+  }, []);
+
+  //fetch GET messages from the API, set messages and catch errors
+  const fetchMessages = () => {
+    fetch(MESSAGES_URL)
+      .then(results => results.json())
+      .then(data => {
+        const filteredData = data.filter(post => {
+          return post.message;
```

Comment:
This method doesn't do anything, because all of elements of this array have `message` property (so at the end `filteredData` and `data` are exactly same arrays). You probably wanted to use `.map()` method